.gitignore

```diff
@@ -27,3 +27,5 @@ hie.yaml
 .direnv/
 result
 lib/cardano-ledger-specs/fine-types
+lib/fine-types-test/test/Language/FineTypes/Export/Haskell/HaskellUTxO.hs
+lib/fine-types-test/test/Language/FineTypes/Export/Haskell/HaskellUTxO1.hs
```

Comment:
```suggestion
```

🤔 Could be very surprising in some distant future. But probably fine — up to you.